### ISSUE TITLE
Broken pretty print on windows

### ISSUE THREAD

**Issue by tarrow** [ContentMine/getpapers#80]
At some point the pretty colour print, loading bar etc.. has stopped working on my windows git bash terminal.

You can see from the screenshot.

![image](https://cloud.githubusercontent.com/assets/7353563/13316267/16f7e9c0-dba7-11e5-92ec-440a957e2c86.png)

**Comment by Blahah** [comment]
I don't have access to a windows machine so can't be of much help here - but if someone who does would like to debug (e.g. by identifying which commit broke this), I'm happy to try to help.

**Comment by tarrow** [comment]
I don't think this is a getpapers issue. I've popped it in because I think other users will hit the same problems however it may be upstream (chalk). Or even with node and how it handles ttys (or things that look like ttys but actually aren't) see: nodejs/node#3006
